doc/classes/BaseMaterial3D.xml

```diff
@@ -177,6 +177,9 @@
 		<member name="disable_ambient_light" type="bool" setter="set_flag" getter="get_flag" default="false">
 			If [code]true[/code], the object receives no ambient light.
 		</member>
+		<member name="disable_decals" type="bool" setter="set_flag" getter="get_flag" default="false">
+			If [code]true[/code], [Decal]s will not display on this material. This is a per-material alternative to [member Decal.cull_mask] which works on a per-object basis. [member disable_decals] can be useful for objects composed of multiple materials where some of these materials should not receive decals (for instance, eyes on a character).
```

Comment:
I will eradicate every "for instance" I see.
```suggestion
			If [code]true[/code], [Decal] nodes do not display on surfaces using this material. This can be seen as an alternative to [member Decal.cull_mask], which works by render layers.
			Enabling this property can be useful for objects composed of multiple materials, as some materials (such as eyes on a character) should never receive decals.
```